### DISCUSSION TITLE
Switch 2 Button

### DISCUSSION THREAD

**Discussion by grillomuto** [arachnetech/homebridge-mqttthing#571] [Q&A]
Hi, can someone help me with the Xiaomi / Aqara QBKG24LM switch https://bit.ly/3vslrgg: This is the Json configuration I am using, it works perfectly but both relays turn on, I can't get them to work separately ... . Thank you   :      {
            "accessory": "mqttthing",
            "type": "switch",
            "name": "Valvola",
            "url": "http://192.168.1.17:1883/",
            "username": "xxxxxxxxxxx",
            "password": "xxxxxxxxxxx",
            "integerValue": "true",
            "topics": {
                "getOn": {
                    "topic": "tele/Nodotubo/SENSOR",
                    "apply": "return JSON.parse(message).ZbReceived['0xD436'].Power;"
                },
                "setOn": {
                    "topic": "cmnd/Nodotubo/ZbSend",
                    "apply": "return '{"Device":"0xD436", "send":{"Power":' + message + '}}'"
                }
            }
        }

**Comment by grillomuto** [comment]
no one deigns to help .....

**Comment by arachnetech** [comment]
We get a lot of requests for configuration help logged as ‘issues’. You’re more likely to get an answer if you show the MQTT messages you’re wanting mqttthing to send/receive in addition to your configuration. 

I very seldom have time to get involved in configuration-related queries but sometimes other helpful people do.

**Reply by tgoschuetz** [reply]
I have a similar problem: I want to publish a fixed message (i.e. not using any switch 'value') when a stateless switch is pressed but I cannot even get the plugin to accept this apply line:

"apply" : "return '{"name":"TestNotification","force":true,"icon":217,"moveIcon":true,"data":"Message text","color":[0,255,0]}'"

The quotation marks I need seem to be the problem. In your example, it seems to work, but I cannot even save a configuration like this.
